pdns/dnsdistdist/libssl.cc

```diff
@@ -53,31 +53,25 @@ static void openssl_thread_cleanup()
 
   OPENSSL_free(openssllocks);
 }
-
-#else
-static void openssl_thread_setup()
-{
-}
-
-static void openssl_thread_cleanup()
-{
-}
 #endif /* (OPENSSL_VERSION_NUMBER < 0x1010000fL || defined LIBRESSL_VERSION_NUMBER) */
 
 static std::atomic<uint64_t> s_users;
```

Comment:
Would you mind moving the `#endif` after the `static std::atomic<uint64_t> s_users;` line, since we don't need `s_users` at all when OpenSSL >= 1.1.0 is in use?